tldw_Server_API/app/core/Ingestion_Media_Processing/persistence.py

```
@@ -0,0 +1,1709 @@
+            try:
+                if saved_files_info:
+                    total_uploaded_bytes = 0
+                    for pf in saved_files_info:
+                        try:
+                            # This mirrors the legacy behaviour, including its
+                            # reliance on Path from FastAPI (errors are ignored).
+                            total_uploaded_bytes += Path(  # type: ignore[arg-type]
+                                str(pf["path"]).strip()
+                            ).stat().st_size
+                        except Exception:
+                            pass
```

Ruff (0.14.4)

`try`-`except`-`pass` detected, consider logging the exception

(S110)